license-information-windows.md

```diff
@@ -2,30 +2,29 @@
 
 The Windows distribution of .NET contains files that are provided under multiple licenses.
 This information is provided to help you understand the license terms that
-apply to your use. By using any or all of these files you agree to their associated license terms.
-
-See [license information](./license-information.md) for information about other operating systems.
+apply to your use. By using .NET on Windows, you agree to the following license terms.
 
 This document is provided for informative purposes only, and is not itself a license.
 
-The following binaries are licensed with the
-[Windows SDK License](https://learn.microsoft.com/legal/windows-sdk/license):
-
-* api-ms-\*.\* (used by .NET runtime, .NET 6 and earlier)
-* ucrtbase.dll (used by .NET runtime, .NET 6 and earlier)
-* D3DCompiler_47_cor3.dll (used by WPF)
-
 The following binaries are licensed with the
 [.NET Library License](https://dotnet.microsoft.com/dotnet_library_license.htm)
 
+* coreclr.dll and .NET runtimes included in binaries published as single-file (due to [extra telemetry](https://github.com/dotnet/runtime/blob/main/src/coreclr/vm/dwreport.cpp) included by .NET runtime in Windows Error Reporting crash reports)
```

Comment:
Why? The linked `dwreport.cpp` file seems to be MIT-licensed?

Reply:
The code itself is MIT. The resultant binary is licensed as .NET Library license because of data collection, per the comment. .NET app crashes on Windows generate memory dumps that get collected by Watson. Those dumps are used to fix bugs in .NET. Many Microsoft products are integrated with Watson. I don't know how many product fixes are made based on Watson information but it has to be a lot. .NET Framework is the same.

There is a clause in the .NET Library License about data collection. The .NET SDK is the other component that I'm aware of that relies on this clause. One can certainly imagine using MIT with software that has a data collection scenario. We decided not to do that.

I remember using Office when I was a kid and seeing the "Dr. Watson" dialog. Yes, Sherlock Holmes themed. Office was the originator of Watson.